src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -73,7 +74,8 @@ private static bool EndsWithPeriodOrSpace(string path)
         /// away from paths during normalization, but if we see such a path at this point it should be
         /// normalized and has retained the final characters. (Typically from one of the *Info classes)
         /// </summary>
-        internal static string EnsureExtendedPrefixIfNeeded(string path)
+        /// TODO: add atribute [return: NotNullIfNotNull("path")]
```

Comment:
Why can't you add it now?

Reply:
Even I have imported using System.Diagnostics.CodeAnalysis; somehow compiler not finding it, 

> D:\dotnet\corefx\src\Common\src\System\IO\PathInternal.Windows.cs(79,18): error CS0246: The type or namespace name 'NotNullIfNotNullAttribute' could not be found (are you missing a using directive or an assembly reference?) 

I guess it because its not included in the project and the Common lib doesn't have project file I could add reference

Reply:
what csproj is failing?

Reply:
corefx\src\System.IO.Ports\src\System.IO.Ports.csproj. JFYI this project is not listed in @stephentoub's list as it depend on netstandard

Reply:
Cool, just curious, thanks.